src/views/Project/ProjectDetail.vue

```diff
@@ -240,25 +242,12 @@
 			)
 		},
 		computed: {
-			auditCondition(){
-				let condition = false
-				if(this.$store.state.auth.user.appRole.name == 'SUPER_ADMIN' || this.$store.state.auth.user.appRole.name == 'ADMIN'){
-					condition = true
-				}
-				else if(this.currentUserRole == 'PROJECT_MANAGER' || this.currentUserRole
-				 == 'PROJECT_USER'){
-					condition = true
-				}
-				return condition;
-			},
-
 			managerCondition(){
 				let condition = false
-				if(this.$store.state.auth.user.appRole.name == 'SUPER_ADMIN' || this.$store.state.auth.user.appRole.name == 'ADMIN'){
-					condition = true
-				}
-				else if(this.currentUserRole == 'PROJECT_MANAGER'){
+				if(this.$store.state.auth.user.appRole.overrideContractRole.authorities.find( authority => authority.name === 'INVITE_MEMBER')){
```

Comment:
use condition = this.$store.state.auth.user.appRole.overrideContractRole.authorities.find( authority => authority.name === 'INVITE_MEMBER')